### PR TITLE
#16 Update code to be compatible with VSF 1.11+

### PR DESCRIPTION
**Pull request by cewald**
This is related to #16.

Maybe it should be a new branch or been tagged to let the old version be still available for users using VSF 1.10-.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,19 +46,17 @@ cart": {
...
 ````js
-import { ExternalCheckout } from './external-checkout'
-// import { InstantCheckout } from './instant-checkout'
-
-export const registerModules: VueStorefrontModule[] = [
-   // other extensions
-  ExternalCheckout,
-  // InstantCheckout,
-]
+import { ExternalCheckout } from 'external-checkout'
```
Comment (glevhen) [inline README.md]: @cewald please restore previous module path value, should be
`import { ExternalCheckout } from './external-checkout'`
path `'external-checkout'` is incorrect and will cause error during installation process

File: index.ts [hunk 2]
```diff
@@ -1,20 +1,13 @@
...
+import { beforeEach } from './router/beforeEach'
+import { routes } from './router/routes'
 
 export const KEY = 'external-checkout'
 
-const moduleConfig: VueStorefrontModuleConfig = {
-  key: KEY,
-  store: {modules: [{key: KEY, module: store}]},
-  router: {beforeEach, routes}
+export const ExternalCheckout: StorefrontModule = function (app, store, router, moduleConfig, appConfig) {
```
Comment (glevhen) [inline index.ts]: @cewald pass  all parameters to registerModule as one object, so replace this line by
`export const ExternalCheckout: StorefrontModule = function ({app, store, router, moduleConfig, appConfig}) {`

**Review by glevhen — CHANGES_REQUESTED** [review]

**Comment by Tjitse-E** [comment]
@cewald can you change your PR so we can merge this?

**Comment by cewald** [comment]
Done :) Sorry for the massive delay.

**Review by glevhen — APPROVED** [review]

**Pull request merged**